Dockerfile.rhel

```diff
@@ -15,6 +15,12 @@ RUN set -x; \
     && yum clean all \
     && rm -rf /var/cache/yum
 
+# Workaround to RHEL8 not having python by default The current version (328) of the
+# Presto fork uses a python script for it's launcher program, which is supposedly
+# python2 and python3 compatible.
+RUN alternatives --set python /usr/bin/python3
```

Comment:
huh had to lookup the alternatives system. This is basically the symlink.approach. Seems good.

Reply:
Yeah I included some of the output of that command locally in one of the commit messages for our future selves.